app/controllers/ems_infra_dashboard_controller.rb

```diff
@@ -0,0 +1,36 @@
+class EmsInfraDashboardController < ApplicationController
+  extend ActiveSupport::Concern
+
+  before_action :check_privileges
+  before_action :get_session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def show
+    if params[:id].nil?
+      @breadcrumbs.clear
+    end
+  end
+
+  def index
+    redirect_to :action => 'show'
+  end
```

Comment:
@h-kataria : why do we need both `show` and `index`. I get that it's a legacy stuff on some controllers but why does it have to be in new ones?